qutebrowser/browser/browsertab.py

```diff
@@ -146,10 +150,33 @@ def run_string(self, name):
             raise WebTabError("{} is not a valid web action!".format(name))
         self._widget.triggerPageAction(member)
 
-    def show_source(self):
+    def show_source(self, pygment=False):
```

Comment:
Given that the library is called `pygments` and not `pygment`, shouldn't this be `pygments`?